service/src/main/java/bio/terra/pipelines/stairway/SendJobSucceededNotificationStep.java

```diff
@@ -0,0 +1,49 @@
+package bio.terra.pipelines.stairway;
+
+import bio.terra.pipelines.common.utils.FlightUtils;
+import bio.terra.pipelines.dependencies.stairway.JobMapKeys;
+import bio.terra.pipelines.notifications.NotificationService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Step to send an email notification that a job has succeeded. This step cannot fail (we catch all
```

Comment:
can we add the expectations from the working map and stuff like we do for other steps (i know its nothing but easier just to read it in the description)